packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -43,6 +43,8 @@ const KOVAN = {
   // https://docs.tokenbridge.net/kovan-sokol-amb-bridge/about-the-kovan-sokol-amb shows 1 for finalization rate
   // but making this the same as mainnet so that the dev experience matches prod
   ambFinalizationRate: '20',
+  relayServiceURL: 'http://localhost:8888/api',
+  subgraphURL: '',
```

Comment:
@lukemelia @pcjun97 can we have relay service and subgraph for kovan?

Reply:
this is possible, but it's not an insignificant amount of work--likely this would need a separate project on its own to setup

Reply:
To confirm, we have decided to use Goerli for this, so we'll need to configure that in a future PR. We will need a relay server but not a subgraph for now.

Reply:
Correct, we need to configure this after we have relay server for goerli